lua/lspconfig/prismals.lua

```diff
@@ -0,0 +1,31 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = "prismals"
+local bin_name = "prisma-language-server"
+if vim.fn.has('win32') == 1 then
+  bin_name = bin_name..".cmd"
+end
+
+configs[server_name] = {
+  default_config = {
+    cmd = {bin_name, "--stdio"};
+    filetypes = {"prisma"};
+    settings = {
+      prisma = {
+        prismaFmtBinPath = "";
+      }
+    };
+    root_dir =util.root_pattern(".git");
```

Comment:
Are there any other relevant files that can act as root directory? For example `package.json` or some Prisma specific configuration file?

Reply:
i added package.json. From what i can tell, there isnt another root file that would be useful.